### DISCUSSION TITLE
🖼 Fanart Competition #4

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2898] [Art]
For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here:

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme
The theme for this month is: **Buds**   ![image](https://github.com/BirbSunflower/SunflowerLand/assets/123522693/6c61dec8-f533-47b5-9456-9060ca95240a)

"After the long wait, Sunflower Land is releasing its first NFT drop! 
Introducing Buds, the friendly farming pets that will turbo charge your Sunflower Land farm and gaming experience. These buds mark the start of a new era of collectibles and utility in Sunflower Land, providing exciting gameplay avenues. Players will now be able to collect a one of a kind Sunflower Land collectible and use it in their farming adventure.
In late September, we will be launching the Bud NFT drop. There will be a collection of 5,000 buds offering a range of perks and in-game boosts."

You can read more about buds [here](https://docs.sunflower-land.com/player-guides/bud-nfts)

![image](https://github.com/BirbSunflower/SunflowerLand/assets/123522693/da1a4567-5dfa-4f7f-ac26-7ba94577470c)

Show us your creativity by designing something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 
- A Google Drive link with proof of your creation. 
To verify the authenticity of your artwork, we kindly request that you provide proof of your creative process. This may include work-in-progress images, concept sketches, time-lapse videos, and, in the case of digital art, we also request the original files.

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆
The winner will receive a Bud and a runner-up will receive 1000 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅
Artists have until September 29th to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

**Comment by Asterion1989** [comment]
Farm: 55626
Discord: jeansb11
![bud_fishing](https://github.com/sunflower-land/sunflower-land/assets/78221179/85b8d8f3-0b64-4778-b3f6-cfe541073b70)

![0001-0045_h8mukac2](https://github.com/sunflower-land/sunflower-land/assets/78221179/2b7428af-ef7f-4340-be78-cc86685a67ec)

Google Drive
https://drive.google.com/drive/folders/1p8musj6aSrK5LXfdGkugLYn9fMMkjkhh?usp=sharing

**Comment by IceMonky29** [comment]
![Finalizado](https://github.com/sunflower-land/sunflower-land/assets/142521805/4e5f17ab-0432-4e1e-9d99-a0ac8d08ee97)

I present to you the giant old bud, along with the other buds, this is a wise and ancient bud that shares his wisdom with everyone who cares for animals and plants, this is my work, titled BUDS LAND!!

---OLD BUD---
![Old Bud](https://github.com/sunflower-land/sunflower-land/assets/142521805/4dfb9fd5-6ef9-4132-a2a6-3e9da1055dfd)
-----

I leave attached a link to Google Drive with the evidence of the creation process.

https://drive.google.com/drive/folders/161eCnEqySd3Iw3xjQCDnzFvHrYyQ7TCA?usp=sharing

Farm: 162510
DISCORD: IceMonky

**Comment by alikotigr** [comment]
Hello dear farmers! 🍀 
Here's my piece of artwork, representing my vision of Buds Fairy World 🌷 
Bright, colorful and full of magic of being a true Bumpkins' helper in becoming a powerful farmer 💪

Discord: thet1ger
Farm: 129970

Google Drive: https://drive.google.com/drive/folders/1tObvzL5_f7CEq9idY_qxr4e_7AHe63kj?usp=sharing
(Hope it works, if not, please DM me, never shared files before).

![Day4-(final)](https://github.com/sunflower-land/sunflower-land/assets/103964833/aea5aeaf-9bae-4adc-ac36-3f65ac0309ea)

**Comment by Vergelsxtn** [comment]
### A helping hand
A bumpkin that taking care of his planted buds is also taking care by his lovely and already harvested bud! what a great companion 😍

(zoom-in for better view)
![fanart-vergelsxtn-github-prev](https://github.com/sunflower-land/sunflower-land/assets/99605039/89a3e41c-8f90-4137-9af2-3c3f83722c3a)

Farm ID: 63
Discord ID: Vergelsxtn

if you want to see the version without blur & wip images you can find it here:
https://drive.google.com/drive/folders/10re2fxVsFR2v9d0ZAmgB3kMBP6ATwsEl

**Comment by andand0** [comment]
### Bud's Adventures

<br>

**Be careful with mushrooms**

My fist Comic strip, it's a short story that I came up with during the Bud's Hats Competition.
I think it would be fun to make some short stories where the buds have funny adventures 

<br>

`Farm ID: 165810`
`Discord: Andando` 

<hr>

Google drive: https://drive.google.com/drive/folders/1dZt8RspNEZDXSdM2xeL5edCVikAhznnJ?usp=sharing

![Bud's Adventures Fanart andando v3](https://github.com/sunflower-land/sunflower-land/assets/142132586/13e1a226-704b-4654-98ee-d6e1fad762d7)

<hr> <hr>

### Cute Bud Animation

<br>


If someone has doubts about having a Bud, just show them this picture. Adorable, right?

<br>


![cute-cute-bud small resolution](https://github.com/sunflower-land/sunflower-land/assets/142132586/bcd8dc56-d499-4de4-9dc7-6a4d12bd44ee)


**High resolution**

![cute-cute-bud hig resolution](https://github.com/sunflower-land/sunflower-land/assets/142132586/61b54e8b-f452-4a63-8c73-d6e5b2f1aa2e)



Google drive: https://drive.google.com/drive/folders/1DW_1fh3mfIii_6Yd8Ih3ZzLIfuUuBRfP?usp=sharing

`Farm ID: 165810`
`Discord: Andando` 

<br>

My NPC
| Gif| Sprite |
|--------|--------|
| ![Andando-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/a4d971a0-dd0f-402f-9471-40c73ceab030)| ![Andando-NPC-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/db596a90-e19a-4cc7-a08f-831f7ef22867)|

**Comment by archila79** [comment]
Hello, this is my contribution.
I hope you like it...
![pixil-frame-0](https://github.com/sunflower-land/sunflower-land/assets/115169880/678f800d-7038-41ac-94c4-769eb9d6161c)

**Reply by archila79** [reply]
farm id:#143991
archila#9474

**Comment by VZozul** [comment]
**This is not about chess, this is about Buds, about the new adventures that await us, gaming rivalries, new gameplay experiences and new opportunities!**

I'm not sure if this type of Fanart will count, but why not.

A little background. At first I thought of the Buds confrontation, Human Buds vs Goblin Buds. One of the thoughts that came to my mind was chess, even though I’m not a fan. I have my own constructor in Figma, so I made it, and then I decided to make a video out of it. The animation may differ, I made it from what I found, but not everything is available.

Watch with sound. Unfortunately, there is no way to add a big video here, so I uploaded it to YouTube.
**https://www.youtube.com/watch?v=3wtkle4YT9c**



![New_Adv](https://github.com/sunflower-land/sunflower-land/assets/103659539/59c99666-f650-4c8c-80e3-ef94c8e54219)

![New_Adv_UP](https://github.com/sunflower-land/sunflower-land/assets/103659539/52d04bab-4762-4191-ab29-e1ac3928fd7f)

**Farm ID: 152456
Discord: VZozul**

Proof - [https://drive.google.com/drive/folders/13UIah1JI9ZkQtYP5mG6aoNSqMfsqOMh1?usp=drive_link](https://drive.google.com/drive/folders/13UIah1JI9ZkQtYP5mG6aoNSqMfsqOMh1?usp=drive_link&authuser=0)

P.S.
It was difficult, I will never edit a video in Photoshop again, on the one hand it’s convenient, when there are a lot of GIFs and you need to make a video out of them, I couldn’t think of a better option. But, as it turned out, Photoshop, with a large number of gifs and elements, does not even allow me to watch the video and no, my computer is not weak. So I had to render, open in premiere and focus on the timeline. Render is a different story, I’ll share a little experience, if you suddenly come across this, then in Photoshop when rendering you need to collapse all layers, otherwise with a very large file, it will simply give an error in the end or simply close the program. Until I found this information, I went through all the agony of hell trying to render it.
Share your experience, maybe there are other programs that can make this easier?

**Reply by alikotigr** [reply]
That's very creative, I like the idea and storyline :)

**Comment by LTNGx39** [comment]
### <h1> Little and cute buds!

```
Farm ID: 161769
Discord: ncnlx39
```

Hello, now I have decided to experiment a little more by uniting 2 different types of art, with a design of the buds from the plaza waiting for their future owners, I hope you like it and good luck to all the participants.

> Folder with process: https://drive.google.com/drive/folders/1LvODFFQvRSRczUAGVsTvsa7S_NUhlvSU?usp=sharing
> There is also a higher resolution version of the fan art there

![Final copia](https://github.com/sunflower-land/sunflower-land/assets/131133076/d1bab913-4806-453a-9125-351e14ad1d26)

**Comment by holyshit89** [comment]
Los nuevos visitantes
The new visitors
![6](https://github.com/sunflower-land/sunflower-land/assets/141026573/ae810333-9fd6-4366-a94c-5c906de336b5)


Farm ID: 159682
Discord: holyshit89

Proceso de trabajo:
https://drive.google.com/drive/folders/11TBjPc0dFi6SOtC6rsAEIt-B8BiBYq7W

**Reply by Leonel619** [reply]
This drawing is beautiful, it reminds me of Harvest Moon, great work by my friend❤❤❤

**Reply by holyshit89** [reply]
Gracias Bro...✌️

**Comment by Scka-lez** [comment]
Farm ID: 129125
DC ID: sckalez
File: https://drive.google.com/file/d/1pGbu-AoOiiNZ1v73mbiWXLy1kavIwasV/view?usp=sharing

![BUDS FANART - Sckalez](https://github.com/sunflower-land/sunflower-land/assets/129434322/410367e5-f83f-4af5-bfc5-c70280bff1d6)

**Comment by peperol** [comment]
A moment of relaxation with my new friends...the pleasures of life in Sunflower land

![IMG_0365](https://github.com/sunflower-land/sunflower-land/assets/114354077/485301e2-a61e-491e-a66d-0312358b76e6)


Farm Id 145849
 Discord peperol

**Comment by shinooon** [comment]
Farm ID: 51738
Discord ID: shinon4931
Fanart
![Buds](https://github.com/sunflower-land/sunflower-land/assets/109294090/557405a1-3059-4193-99ab-927e61620b8e)
NPC:
![my npc_mitsuha](https://github.com/sunflower-land/sunflower-land/assets/109294090/d08b32c7-3bef-447a-96ad-d86eba646e1b)
GDrive Link: https://drive.google.com/drive/folders/17x6S4F-ATv0cN6MXwP-gKk3WlWY7RATg?usp=drive_link
